jobs_detector/main.py

```diff
@@ -1,33 +1,97 @@
-import click
-import requests
+import click, re, requests
 from bs4 import BeautifulSoup
+# from jobs_detector import settings
+import settings, exceptions
+from helpers import _keyword_search, _check_combinations, _get_percent
+from collections import defaultdict
 
-from jobs_detector import settings
-
-DEFAULT_KEYWORDS = [
-    # set some default keywords here
-]
+@jobs_detector.command()
+@click.option('-i', '--post-id', type=str, required=True, help= 'HN Post ID.')
+@click.option('-k', '--keywords', type=str, default=','.join(DEFAULT_KEYWORDS),
+help= "Specify keywords")
+@click.option('-c', '--combinations', type=str, help= "Specify keyword combinations to match",
+              callback=lambda _, x: x.split(',') if x else x)
+@click.argument('hacker_news')
```

Comment:
I'm not exactly sure how @click.argument works. I thought the @click.option decorator had to be above the def hacker_news line so that it would decorate that function. It looks like you moved some stuff around from the original template.